MyChat/ConversationExporterConfiguration.cs

```diff
@@ -14,5 +14,29 @@ public sealed class ConversationExporterConfiguration
         /// The output file path.
         /// </summary>
         public string OutputFilePath { get; set; }
+
+        /// <summary>
+        /// Filters the results by a user.
+        /// Example: --filterByUser bob,angus
+        /// </summary>
+        public string FilterByUser { get; set; }
+
+        /// <summary>
+        /// Filters the results by a keyword.
+        /// Example: --filterByKeyword pie,no
+        /// </summary>
+        public string FilterByKeyword { get; set; }
+
+        /// <summary>
+        /// Hides a keyword by replacing it with *redacted*.
+        /// Example: --blacklist hello
+        /// </summary>
+        public string Blacklist { get; set; }
+
+        /// <summary>
+        /// Adds a report of message count per user to the output.
+        /// Example: --report true

```

Comment:
If you input '--report' with no entry, it doesn't get picked up and will always just return false.